src/nl/hannahsten/texifyidea/gutter/LatexNavigationGutter.kt

```diff
@@ -104,8 +108,10 @@ class LatexNavigationGutter : RelatedItemLineMarkerProvider() {
                 .toList()
 
         // Build gutter icon.
-        val maxSize = WindowManagerEx.getInstanceEx().getFrame(element.getProject())?.size?.width
-                ?: return
+        // TODO: do we really need to dynamically find the maxSize? WindowManagerEx doesn't work in unit tests
```

Comment:
I also wondered about this, because getFrame is marked as unstable internal api so I wanted to get rid of it anyway. I can see no adverse effect so I'll remove the todo.